go.mod

```diff
@@ -1,10 +1,9 @@
 module github.com/bwplotka/mdox
 
-go 1.15
+go 1.19
```

Comment:
I think you raised PR for this separately #96?

Reply:
Yeah, this PR is on top of that one. If #96 gets merged, this won't show those commits.
(Without stacking them, there are conflicts because of the dependency updates.)